data/wallets/flow-wallet/flow-wallet.ts

```diff
@@ -0,0 +1,108 @@
+import { nextJsImageToBase64 } from '../../../helpers/image'
+import { WalletConfig } from '../../wallets'
+import FlowWalletIcon from './flow-wallet.png'
+
+const flowWallet: WalletConfig = {
+  name: 'Flow Wallet',
+  uid: 'flow-wallet',
```

Comment:
What are the `uid`s used for now? Can they be deprecated? Maybe not until the old way is sunsetted?